README.md

```diff
@@ -150,3 +150,30 @@ When running the app locally, ensure you are using the development `GOOGLE_SHEET
 (See google drive for this)
 
 If updating your environment variable doesn't work, try `rm -rf _build` and then starting your server again
+
+### Image uploads
+
+##### File size
+
+The Wagtail cms allows for a maximum file size of 10MB.
+
+Althouth I would suggest that files be no bigger than 1MB.
```

Comment:
Not to be picky, but could we make this a little more general rather than personal? 'we' or 'it is recommended that' :)